docs/administration.rst

```diff
@@ -112,3 +133,18 @@ once they have been started, users can use ``bodhi composes list`` or they can v
 * **signing_repo**: The composer is waiting on the repository to be signed.
 * **cleaning**: The composer is cleaning up old composes. This state only occurs if
   ``clean_old_composes`` is set to True in the settings.
+
+
```

Comment:
Greenwave Handler

Bodhi's greenwave handler watches for messages sent from greenwave about bodhi updates that have been tested.  When a message is recieved, the appropriate update is updated with the greenwave test status.

Reply:
I'm adding this patch:

```
diff --git a/docs/administration.rst b/docs/administration.rst
index 1eb4e5ca..3485b585 100644
--- a/docs/administration.rst
+++ b/docs/administration.rst
@@ -135,6 +135,14 @@ once they have been started, users can use ``bodhi composes list`` or they can v
   ``clean_old_composes`` is set to True in the settings.
 
 
+Greenwave Handler
+^^^^^^^^^^^^^^^^^
+
+Bodhi's greenwave handler watches for messages sent from greenwave about bodhi updates that have
+been tested. When a message is recieved, the appropriate update is updated with the greenwave test
+status.
+
```
+
 Signed Handler
 ^^^^^^^^^^^^^^